### DISCUSSION TITLE
Is there any reason why there is no search facility in pester.dev docs?

### DISCUSSION THREAD

**Discussion by plastikfan** [pester/Pester#2051] [Q&A] [answered]
If I am not mistaken, the pester.dev documentation is built with docusaurus, which provides a search faciltiy out of the box. I'm just wondering why it appears to have been disabled. It would help users getting to the right information faster.

**Comment by nohwnd** [comment]
@bravo-kernel I was wondering the same thing. Was that because of the indexing engine not being free, or did we just forgot to add it? 😁

**Comment by bravo-kernel** [comment]
I just recently re-applied, two days ago. Am awaiting green light. Their indexing service is great (and free) but the signup process is a bit of a bummer.

**Comment by bravo-kernel** [comment]
Just chased the request and we should get set up within a couple of days. After that, I will PR the search for pester-docs. 🙏

**Comment by plastikfan** [comment]
Seems like there's more to this than I had realised. Glad to see some wheels spinning ...

**Reply by fflaten** [reply]
It's been in planning phase for years. 😁 pester/docs#4

**Reply by bravo-kernel** [reply]
Including multiple applications iirc 🙄

**Reply by nohwnd** [reply]
How the time flies. I thought we started pester.dev just few months ago. 😮

**Comment by bravo-kernel** [comment, marked as answer]
Suggesting to close this discussion as there now is a PR open: https://github.com/pester/docs/pull/143

**Comment by plastikfan** [comment]
Fine by me

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nohwnd** [comment]
There is one now :)
